examples/browser/test-fetch-pipeline.html

```diff
@@ -0,0 +1,78 @@
+<?xml version = "1.0" encoding="utf-8" ?>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN"

```

Comment:
This document claims to be XHTML 1.0 Strict but does not conform to this standard. Please ensure it passes validation with http://validator.nu/ . You may change DOCTYPE if necessary.